iceoryx_examples/waitset/README.md

```diff
@@ -370,12 +371,11 @@ class SomeClass
     static void cyclicRun(iox::popo::UserTrigger*)
     {
         std::cout << "activation callback\n";
-        trigger->resetTrigger();
     }
 };
 ```
-**Important** We need to reset the user trigger otherwise the _WaitSet_ would notify
-us immediately again since the user trigger is state based.
+**Important** The user trigger is event based and always reset after the WaitSet 
```

Comment:
As this Readme will be included on the website, please use the [MkDocs syntax](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) here:

```
!!! attention
    Foo bar
```